package/network/services/lldpd/files/lldpd.init

```diff
@@ -1,15 +1,23 @@
 #!/bin/sh /etc/rc.common
+# Copyright (C) 2018 Anton Kikin <a.kikin@tano-systems.com>
```

Comment:
Please split this big commit into  multiple commits to make it easier to review the changes.

Reply:
Mhh, I don't know much about Git and I didn't write the code myself, so I don't know how far I can do something about it. Alternatively, maybe (part of) the git history could be taken from the TanoWrt repo.

Reply:
How do you want the PR split? This is focussed on lldpd and rewrites the init script so that it respects settings in /etc/config/lldpd correctly, whereas the previous version was ignoring uci setting for lldpd interface bindings.
I can help restructure the PR if you like - I'm interested in seeing this included and know Git. 
@hauke any guidance on acceptable commit sizes? I see there are already 5 commits dealing with `lldpd.init` and the changes are limited to the correct reading/writing of settings from /etc/config/lldpd.

Reply:
Just break it up into random commit sizes until @hauke is satisfied :)

I guess the idea is to attempt to capture some of the original commit messages and any combined wisdom there. If taking the Tanto history, rebase from those commits and slap your changes on top.

Reply:
I think the commits already reflect the decisions made here so there's not a lot to add. It's really a rewrite of a single file so I'm surprised that maintainers wouldn't want this squashed to a single commit when they merge it.
Changes can be summarised as - remove redundant LLDP example location and allow init script to correctly interact with UCI configuration for binding to interfaces and settng management interface when starting LLDPD.

Reply:
Yeah, then it's just to get some approval. 

@jow- @hauke @hnyman ?